testVectors.json

```diff
@@ -579,128 +573,6 @@
       "output": "0x48e5a1a0e616d8fd92b4ef228c424e0c816799a256c6a90892195ccfc53300d6"
     }
   ],
-  "parseInput": [
-    {
```

Comment:
Are we sure we want to delete these test vectors?  It looks like you moved `parseInput` inside `/golang/cli/vin.go` and `parseOutput` inside `/golang/cli/vout.go`.  If those functions are still present somewhere, we should keep these vectors and test them.

Reply:
Yes definitely. The moved code is now officially untested "example" code. That means we don't have to maintain tests or quality. #deletedcodeisdebuggedcode :)